Gemfile

```diff
@@ -2,9 +2,10 @@
 
 source 'https://rubygems.org'
 
-gem 'octocat', '= 0.0.2'
+gem 'octokit', '~> 4.6'
 gem 'libxml-ruby', '> 0'
 gem 'libxslt-ruby', '> 0'
+gem 'licensee', '~> 8.7' # CHK TODO this requires cmake and a bunch of gnarly dependencies, as the comment in README.md mentions, should we just patch octokit (hasn't been updated since 2014) to get this now that we can assume github provides this?
```

Comment:
Following up the CHK TODO: Using the GitHub license API would stop you from being able to have your own license matching locally (I think). Unless they also publish the cksum when they can't identify a license. I had problems getting to it through Octokit at the time and need to see.